pom.xml

```diff
@@ -17,6 +17,8 @@
     <modules>
         <module>scribejava-core</module>
         <module>scribejava-apis</module>
+        <module>scribejava-ahc</module>
+        <module>scribejava-ning</module>
```

Comment:
Sometime ScribeJava will maybe have some new set of submodules.
Let this httpclient submodules pack will be named
scribejava-httpclient-ahc
and
scribejava-httpclient-ning